config/m4/ucg.m4

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (C) Huawei Technologies Co., Ltd. 2019.  ALL RIGHTS RESERVED.
+# Copyright (C) Mellanox Technologies Ltd. 2020.  ALL RIGHTS RESERVED.
+# See file LICENSE for terms.
+#
+
+#
+# Enable UCG - Group collective operations component
+#
+
+ucg_modules=""
+
+m4_sinclude([src/ucg/configure.m4]) # m4_sinclude() silently ignores errors
+
+#
+# TODO this section can be moved inside src/ucg/configure.m4, which will be
+# included only if exists. In this case m4_esyscmd() and m4_if() wrapper can
+# also be removed.
+#
+AC_ARG_ENABLE([ucg],
+              AS_HELP_STRING([--enable-ucg],
+                             [Enable the group collective operations component, default: NO]),
```

Comment:
We have to indicate in the configure help message that this an experimental API/Component.

Reply:
@shamisp - how about the following?

"Enable the group collective operations (experimental component), default: NO"

Reply:
👍